Customer.io Cloud Extract source (#929)
rudderlabs/rudderstack-docs: docs/sources/extract/customerio.mdx

---
title: "Customer.io"
description: Step-by-step guide on ingesting data from Customer.io into RudderStack.
aliases: ["Customer.io source"]
---

> [Customer.io](https://www.Customer.io.com/) is a popular marketing platform for sending targeted emails and push and SMS notifications to improve customer engagement and thereby the overall conversion rate.

Reviewer: ```suggestion
[Customer.io](https://www.Customer.io.com/) is a popular marketing platform for sending targeted emails and push and SMS notifications to improve conversions and customer engagement.
```

This document guides you in setting up Customer.io as a source in RudderStack. Once configured, RudderStack automatically ingests your Customer.io data and routes it to your specified data warehouse destination.

<div class="warningBlock">
All the Cloud Extract sources support sending data only to a <Link to="/destinations/warehouse-destinations/">data warehouse destination</Link>.
</div>

## Getting started

To set up Customer.io as a source in RudderStack, follow these steps:

1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
2. Go to **Sources** > **New source** > **Cloud Extract** and select **Customer.io** from the list of sources.
3. Assign a name to your source and click **Continue**.

### Connection settings

To set up Customer.io as a Cloud Extract source, you need to configure the following settings:

<img src="../../assets/cloud-extract-sources/customerio-connection-settings.png" alt="Customer.io credentials" />

- **App API Key**: Enter your Customer.io API key which can be obtained in the [Customer.io dashboard](https://fly.customer.io/login) by navigating to **Settings** > **Account Settings** > **API Credentials**.

> - **Cutoff Days**: Enter the number of days after which the updated data should be fetched.

Reviewer: ```suggestion
- **Cutoff Days**: Enter the number of days after which RudderStack fetches the updated data.
```
